strax/context.py

```diff
@@ -679,17 +679,29 @@ def _find_options(self):
                     fuzzy_for_options=self.context_config['fuzzy_for_options'],
                     allow_incomplete=self.context_config['allow_incomplete'])
 
+    @property
+    def _sorted_storage(self) -> ty.List[strax.StorageFrontend]:
+        """
+        Simple ordering of the storage frontends on the fly when e.g.
+        looking for data. This allows us to use the simple self.storage
+        as a simple list without asking users to keep any particular
+        order in mind. Return the fastest first and try loading from it
+        """
+        storage_sfs = self.storage
+        storage_sfs.sort(key=lambda x: x.storage_type)
```

Comment:
```suggestion
        storage_sfs = sorted(self.storage, key=lambda x: x.storage_type)
```
I think you want to keep original list since order is used in `copy_to_frontend` right? I think the way it is would sort in place the original list no?

Reply:
Nice!